html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Form</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+
+  <form action="" method="get">
+
+    <div>
+      <label for="Email">Email</label>
+      <input type="text" name="Email" id="Email">
+    </div>
+
+    <label for="select an option">Why is coding cool?</label>
+    <div>
+      <select name="select an option" id="select an option">
+        <option value="Select an option">Select an option</option>
+        <option value="Interesting">Because it's Interesting</option>
+        <option value="Fun">Because it's fun</option>
+        <option value="Powerful">Because it's Powerful</option>
+      </select>
+    </div>
+
+    <label for="radio">Favorite Language</label>
+    <div>
+      <div>
+        <label for="HTML">HTML</label>
+        <input type="radio" name="HTML" id="HTML">
+
+        <label for="CSS">CSS</label>
+        <input type="radio" name="CSS" id="CSS">
+
+        <label for="Java">Javascript</label>
+        <input type="radio" name="Java" id="Java">
+      </div>
+
+      <label for="sign">Would you like to sign up for more information?</label>
```

Comment:
This is actually just general text describing the section below since it isn't labeling one specific form control element. You should use a `p` element.